### PR TITLE
getTranslation with bool withFallback

### PR DESCRIPTION
**Pull request by meta0102**
hopefully the tests are working as "composer test" is not found as a command and i've no idea how to run the test files

### CODE REVIEW

File: tests/TranslatableTest.php [hunk 1]
```diff
@@ -1,133 +1,120 @@
 <?php
-
```
Comment (freekmurze) [inline tests/TranslatableTest.php]: Could you please restore all the empty lines you've deleted in this file?

**Comment by meta0102** [comment]
Sorry... sure, will do it tomorrow morning

**Comment by meta0102** [comment]
empty lines restored

**Comment by freekmurze** [comment]
Thank you!

**Pull request merged**